Realm/Tests/RealmTests.mm

```diff
@@ -1172,6 +1173,24 @@ - (void)testHoldRealmAfterSourceThreadIsDestroyed {
     [realm path]; // ensure ARC releases the object after the thread has finished
 }
 
+- (void)testCompact
+{
+    RLMRealm *realm = self.realmWithTestPath;
+    [realm transactionWithBlock:^{
+        [StringObject createInRealm:realm withObject:@[@"A"]];
+        [StringObject createInRealm:realm withObject:@[@"A"]];
```

Comment:
why would we want to test *more* than the minimal case? If it compacts in this minimal case, it will in larger cases too.